seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-kafka/src/main/java/org/apache/seatunnel/flink/kafka/source/KafkaTableStream.java

```diff
@@ -114,7 +114,7 @@ public void prepare(FlinkEnvironment env) {
         try {
             schemaInfo = JsonUtils.stringToJsonNode(schemaContent);
         } catch (JsonProcessingException e) {
-            e.printStackTrace();
+            log.error("schema parse error", e);
```

Comment:
@CalvinKirs  I didn't understand what you meant by  throwing specific exception information yesterday, but now I got it.